src/static/js/collab_client.js

```diff
@@ -178,10 +178,11 @@ const getCollabClient = (ace2editor, serverVars, initialUserInfo, options, _pad)
   };
 
   const sendMessage = (msg) => {
-    getSocket().json.send(
+    getSocket().emit('message',
         {
           type: 'COLLABROOM',
           component: 'pad',
+          padId: clientVars.padId,
```

Comment:
Can you explain why this is needed?

Reply:
As above, in the first round `usersession` does not work properly and I have to send` padId` from the client